api/src/main/java/org/jboss/arquillian/graphene/GrapheneElementAction.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * JBoss, Home of Professional Open Source
+ * Copyright 2013, Red Hat, Inc. and individual contributors
```

Comment:
Technically yes - the license file should be fine - I'll keep it here for now and in next PR I'll remove the check for the presence of the header. 
I'm not sure if the license file should have the name: LICENSE or if it can be as it is now: license-lgpl-2.1.txt